systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -178,10 +186,8 @@ void testTieredStorageWithAivenPlugin() {
                 return earliestLocalOffset > 0;
             });
 
-        ClientUtils.waitForInstantProducerClientSuccess(testStorage);
-
         resourceManager.createResourceWithWait(clients.consumerStrimzi());
-        ClientUtils.waitForInstantConsumerClientSuccess(testStorage);
+        ClientUtils.waitForClientSuccess(testStorage.getNamespaceName(), testStorage.getConsumerName(), MESSAGE_COUNT);
```

Comment:
1. Because we have produced 10000 messages in L159 above, we don't need to produce messages again.
2. The consumer should verify it can consume messages from (a) remote storage and (b) local storage. Because we have verified earlier that the log segments are deleted locally (by earliest-local offset check) and moved to remote storage, we can verify it by checking if we can consume 10000 messages successfully.